radio/src/opentx.h

```diff
@@ -872,14 +872,14 @@ void doLoopCommonActions();
 
 #define BITMASK(bit) (1<<(bit))
 
-/// liefert Dimension eines Arrays
+/// retruns the number of elements of an array
 #define DIM(arr) (sizeof((arr))/sizeof((arr)[0]))
 
-/// liefert Betrag des Arguments
+/// AVR compîler does not optimize 'abs' well
+#if defined(__AVR_LIBC_MAJOR__)
```

Comment:
Is it really needed?
I have a 4bytes flash saving if I remove the line on AVR (tested on 9X stock)